integration/raft/config_test.go

```diff
@@ -162,7 +162,7 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 			By("Starting orderer with malformed genesis block")
 			ordererRunner := network.OrdererGroupRunner()
 			process := ifrit.Invoke(ordererRunner)
-			Eventually(process.Wait, network.EventuallyTimeout).Should(Receive()) // orderer process should exit
+			Eventually(process.Wait(), network.EventuallyTimeout).Should(Receive()) // orderer process should exit
```

Comment:
I know it's a dead giveaway when in any other place in the code, we do `Wait()` and not `Wait`, but, I couldn't just push a PR without explaining why the current code is wrong.